cfme/utils/appliance/__init__.py

```diff
@@ -2067,7 +2075,7 @@ def enable_embedded_ansible_role(self):
             self.server_roles = roles
         except TimedOutError:
             wait_for(lambda: self.server_roles == roles, num_sec=600, delay=15)
-        self.wait_for_embedded_ansible()
+        self.wait_for_embedded_ansible(2400)
```

Comment:
Do we still this this much time for Embedded Ansible to enable? @sbulage ?

Reply:
@kedark3 yes for CF version below 5.11

Reply:
@jaryn can you please decide the wait time using version info? Check Sbulage's comment above